src/pocketmine/Player.php

```diff
@@ -1981,8 +1981,8 @@ public function onVerifyCompleted(LoginPacket $packet, ?string $error, bool $sig
 	protected function processLogin(){
 		foreach($this->server->getLoggedInPlayers() as $p){
 			if($p !== $this and ($p->iusername === $this->iusername or $this->getUniqueId()->equals($p->getUniqueId()))){
-				if(!$p->kick("logged in from another location")){
-					$this->close($this->getLeaveMessage(), "Logged in from another location");
+				if(!$p->kick("disconnectionScreen.loggedinOtherLocation")){ // "Logged in from other location"
+					$this->close($this->getLeaveMessage(), "disconnectionScreen.serverIdConflict"); // "Cannot join world. The account you are signed in to is currently playing in this world on a different device."
```

Comment:
I don't think this is suitable. This might be fired because of two completely different players with the same username.